Adding support for PTP Compatible Digital Camera's

toys4me opened the pull request:
- libgphoto2
- gphotofs
- udev rules for PTP camera's
- script's to auto mount/umount camera

When plugged in a compatible camera will mount as /media/camera
Can then browse and view photo's/videos via file system or through xbmc picture or video source.

davilla commented:
@theuni , please review

theuni commented:
Thanks for the contribution :)

Please see individual comments in-line.

davilla commented:
@theuni , inject ?

theuni commented:
Once the manual file is automated, this can be squashed and merged. It'd be really nice to see separate commits for the 2 packages (hint ;)

toys4me commented:
Will have to go back and play around with getting the host version to build properly for the udev script generation.  As never got that to work right. 

So after I figure out the manual config stuff:
I should try to split the two packages out into seperate branches, squash them, close this pull, and submit them against each as new seperate pull requests?
(I understand git pretty well, just am a complete novice at github)

toys4me commented:
Oh, I already have a version with this moved to thirdparty/ directory, just haven't got it pushed up yet.

theuni commented:
Nah, just break them up into 2 commits and force-push them to this branch.

toys4me commented:
I knew I was making things to difficult on myself.   :) lol

Will see if I can find some time to work on this in the next few days.

toys4me committed:
Add libgphoto2 package to thirdparty packages

libgphoto2 package gives ability to access PTP/MTP compatible camera's and devices

toys4me committed:
Add gphotofs package - file system support for PTP Cameras
- gphotofs
- udev rules for PTP camera's
- script's to auto mount/umount camera

When plugged in a compatible camera will mount as /media/camera
Can then browse and view photo's/videos via file system
